Project-overview.md

```diff
@@ -0,0 +1,99 @@
+This document describes the project to make "CSS short descriptions" available to third-party tools such as code editors or developer tools.
+
+The "short description" is the opening sentence or two of a CSS property reference page. It gives a very short overview of the property. It's also known as a summary.
```

Comment:
A scope question: are short descriptions only ever going to be for CSS properties? Or might we someday expand to at-rules or selectors or something?

Reply:
It will probably expand (and probably not just to CSS). But let's keep it like this for now.